### ISSUE TITLE
Spyder Error

### ISSUE THREAD

**Issue by vimalkum** [ContinuumIO/anaconda-issues#4699]
## Main error
Application <b>spyder</b> launch may have produced errors.
## Traceback
```
Traceback (most recent call last):
  File "/home/harshitha/anaconda3/bin/spyder", line 6, in <module>
    sys.exit(spyder.app.start.main())
  File "/home/harshitha/anaconda3/lib/python3.5/site-packages/spyder/app/start.py", line 103, in main
    from spyder.app import mainwindow
  File "/home/harshitha/anaconda3/lib/python3.5/site-packages/spyder/app/mainwindow.py", line 165, in <module>
    from spyder.utils.environ import WinUserEnvDialog
  File "/home/harshitha/anaconda3/lib/python3.5/site-packages/spyder/utils/environ.py", line 20, in <module>
    from spyder.widgets.variableexplorer.collectionseditor import CollectionsEditor
  File "/home/harshitha/anaconda3/lib/python3.5/site-packages/spyder/widgets/variableexplorer/collectionseditor.py", line 46, in <module>
    from spyder.widgets.variableexplorer.importwizard import ImportWizard
  File "/home/harshitha/anaconda3/lib/python3.5/site-packages/spyder/widgets/variableexplorer/importwizard.py", line 26, in <module>
    import pandas as pd
  File "/home/harshitha/anaconda3/lib/python3.5/site-packages/pandas/__init__.py", line 51, in <module>
    plot_params = pandas.plotting._style._Options(deprecated=True)
AttributeError: module 'pandas' has no attribute 'plotting'

```
## System information
```
version: 1.6.2
qt: 5.6.2
os: Linux;4.4.0-92-generic;#115-Ubuntu SMP Thu Aug 10 09:04:33 UTC 2017;x86_64;x86_64
conda: 4.3.25
platform: linux-64
pyqt: 5.6.0
language: en_IN
python: 3.5.3
```

**Comment by goanpeca** [comment]
@ccordoba12 ?

**Comment by ccordoba12** [comment]
This was fixed in Spyder 3.2.1, so please update to that version.
